### DISCUSSION TITLE
这是啥问题？

### DISCUSSION THREAD

**Discussion by Hi-MrZeFr** [spencerwooo/onedrive-vercel-index#274] [Q&A] [answered]
{"error":{"error":{"code":"itemNotFound","message":"The resource could not be found.","innerError":{"date":"2022-01-14T09:29:32","request-id":"7fee4456-abd6-4258-aeb0-4f03ee43bd13","client-request-id":"7fee4456-abd6-4258-aeb0-4f03ee43bd13"}}}}

**Comment by spencerwooo** [comment, marked as answer]
根目录文件夹名写错了，baseDirectory 那里

https://docs.spencerwoo.com/getting-started#60c495e6195b4cee825fdffc3e388288

![image](https://user-images.githubusercontent.com/32114380/149499151-655078f5-b670-4eb1-a54d-bad38f39b3a4.png)

**Reply by Hi-MrZeFr** [reply]
谢谢

**Reply by Hi-MrZeFr** [reply]
什么时候能支持上传和后台管理呢？

**Reply by spencerwooo** [reply]
有朝一日吧，上传不会支持的

**Reply by Hi-MrZeFr** [reply]
为啥全是英文呢？

**Reply by Hi-MrZeFr** [reply]
搞中文吧
反正都是中国人

**Reply by Hi-MrZeFr** [reply]
或者搞个语言切换也行

**Reply by Hi-MrZeFr** [reply]
或者支持一下壁纸
